src/ingest_validation_tools/table-schemas/includes/fields/data_collection_mode.yaml

```diff
@@ -1,2 +1,2 @@
 - name: data_collection_mode
-  description: Mode of data collection in tandem MS assays. Either DDA (Data-dependent acquisition) or DIA (Data-indemendent acquisition.
+  description: Mode of data collection in tandem MS assays. Either DDA (Data-dependent acquisition) or DIA (Data-independent acquisition.
```

Comment:
```suggestion
  description: Mode of data collection in tandem MS assays. Either DDA (Data-dependent acquisition) or DIA (Data-independent acquisition).
```

Reply:
@cebriggs7135 -- I'm going to merge my suggestion and regenerate, but in general, the PR author should follow-up on any comments.